apps/wakunode2/app.nim

```diff
@@ -69,8 +69,11 @@ type
   App* = object
     version: string
     conf: WakuNodeConf
-
+    netConf: NetConfig
     rng: ref HmacDrbgContext
+    nodeKey: crypto.PrivateKey
```

Comment:
I agree but both Discv5 and node require keys for setup that why it's higher in the call chain.

Would calling it appkey helps :P

Reply:
We indeed have an `appKey` if it's used to setup both the discovery and the node. The `appKey` is then used to initialise the (logically distinct) keys for the node and discv5 with the same value.

Reply:
Ok understood :)

However, given that `discv5` should act as a separate service, a good place to start it would be in the next proc:
https://github.com/waku-org/nwaku/blob/a44d4bfbcd0d67bd5920fc85091d2feee8b5a676/apps/wakunode2/app.nim#L763

The key should be generated there from the 'key' info in the config.